packages/fxa-auth-server/lib/senders/emails/templates/downloadSubscription/index.stories.ts

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Meta } from '@storybook/html';
+import { subplatStoryWithProps } from '../../storybook-email';
+
+export default {
+  title: 'SubPlat Emails/Templates/downloadSubscription',
+} as Meta;
+
+const createStory = subplatStoryWithProps(
+  'downloadSubscription',
+  'Sent when a user successfully adds a subscription.',
+  {
+    icon: 'https://accounts-static.cdn.mozilla.net/product-icons/mozilla-vpn-email.png',
+    link: 'http://getfirefox.com/',
+    productName: 'Firefox Fortress',
+    subscriptionSupportUrl: 'http://localhost:3030/support',
+    playStoreLink:
+      'https://play.google.com/store/apps/details?id=org.mozilla.firefox',
+    appStoreLink:
+      'https://apps.apple.com/us/app/firefox-private-safe-browser/id989804926',
+    // Forces fallbacks to be used in appBadges partial
+    iosUrl: undefined,
+    androidUrl: undefined,
```

Comment:
https://github.com/mozilla/fxa/blob/ca61a3fac9d691676e4a6ce5b41c5753cd8f174f/packages/fxa-auth-server/lib/senders/emails/storybook-email.ts#L27

I don't know that we need to explicitly set these to `undefined` but it's OK to leave this as-is.

Sorry for not starting a review, this is my last comment. Thank you for all your work here, this is all super great!! 🙂